### PR TITLE
Extend notes on form_valid exception handling

### PR DESCRIPTION
**Pull request by codeinthehole**
To cover when to catch the base Exception class and how to handle it.

### CODE REVIEW

File: python.md [hunk 1]
```diff
@@ -566,10 +567,31 @@ class CreateSomething(generic.FormView):
...
+            # and fixed. It's optional as we could omit this block and let the
+            # exception percolate up to generate a 500 response which Sentry
+            # will capture automatically. It's a bit friendlier to the user to
+            # show them some custom copy rather than a generic 500 response.
+            #
+            # Depending on context, it might not be appropriate to
+            # show the exception message to the end-user. If possible, offer
+            # some guidance on what the end-user should do next (eg contact
+            # support, try again, etc)
+            msg = "Really sorry, we weren't able to do the thing as an unanticipated error occurred. Error message: %s" % e
```
Comment (seddonym) [inline python.md]: Don't we need to log the exception, too, to make sure it gets to Sentry?

Comment (seddonym) [inline python.md]: This line is a bit long for reading on the markdown page in Github.

Comment (codeinthehole) [reply python.md]: 🤦‍♀️

Comment (codeinthehole) [reply python.md]: Yep - adjusted

**Pull request merged**